posts/2024-01-24_wyg-zooma/README.md

```diff
@@ -0,0 +1,45 @@
+
+## Making Web3 More Accessible 
+
+Of course, as zôÖma found out, building new tools is easier said than done. Our industry faces an uphill climb when it comes to balancing the promise of the tech with a user experience that doesn’t cause tachycardia. He says that understanding that most people “don’t have the time or inclination to incorporate difficult technical concepts in their lives” has given him “crazy energy to focus on very simple technologies.” In fact, the ‘failure’ of La Suite du Monde is what gave birth to Teritori, “which today provides all the functionalities people asked us for at the time; a social network, communication systems, voting, crowd-funding, etc. We have made great progress, and it’s important to focus on products that are radically simple for the general public.”
+
+According to zôÖma, this means abstracting away the concepts that everyday people don’t need to be aware of, such as networks, dApps, and even blockchain, “and always switching from one decentralized application to another.” Unifying (not centralizing) separate tools, networks, and technologies within a single, simple interface, he believes, is the key to broader adoption. “It's a very complex challenge, in terms of security, design, etc., but it's what I'm passionate about today.” 
+
+When it comes to Gno.land, Teritori has already delivered essential DAO tooling and standards, a Moderation DAO module to facilitate social communication and a Justice DAO module for conflict resolution. The team is now focusing on an on-chain project management tool to allow organizations and individuals to manage projects and track tasks smoothly and transparently on-chain.
+
+## A Fairer, More Transparent World
+
+In 2024, Teritori enters a new phase called "Chapter II", which involves unifying all its work into a mobile and desktop application that could “trigger superb demonstrations of the potential of DAOs.” He enthuses, “I dream that we will see the emergence of a village that uses Teritori as a tool for internal discussion and co-financing. Will this be real in 2024? Who knows? But that’s where I focus all my energy!”
+
+He believes the internet has been a great leveler, enabling anyone with a connection to educate themselves on any subject; yet, the opportunity isn’t open to all, and free and open access is constantly diminishing. “I am a child of the internet. I grew up with warez, p2p, and an internet which provided me with daily resources to learn freely, everything that interested me. In some countries, it is impossible to benefit from this opportunity, and with the centralization of the internet on different key players, mass surveillance, and the censorship of certain dictators, the Internet is losing its very essence, which makes it magic. Distributed protocols can reshuffle the cards and offer tools for the public good.” 
```

Comment:
```suggestion
He believes the Internet has been a great leveler, enabling anyone with a connection to educate themselves on any subject; yet, the opportunity isn’t open to all, and free and open access is constantly diminishing. “I am a child of the internet. I grew up with warez, p2p, and an Internet which provided me with daily resources to learn freely, everything that interested me. In some countries, it is impossible to benefit from this opportunity, and with the centralization of the Internet on different key players, mass surveillance, and the censorship of certain dictators, the Internet is losing its very essence, which makes it magic. Distributed protocols can reshuffle the cards and offer tools for the public good.” 
```

We should be consistent with the way we capitalize/don't capitalize terms.

Reply:
Agreed. I prefer to have it all lower-case, then. No one writes Internet with a capital I anymore; it's like writing out the world wide web.

Reply:
Sure, agree with consistency, but I prefer to keep to lowercase "i". I will change it. Thanks!